Documentation/observability/metrics.rst

```diff
@@ -664,7 +670,43 @@ Labels
 - ``method`` is the HTTP method of the request/response.
 - ``protocol`` is the HTTP protocol of the request, (For example: ``HTTP/1.1``, ``HTTP/2``).
 - ``status`` is the HTTP status code of the response.
-- ``reporter`` identifies the origin of the request/response. It is set to ``client`` if it originated from the client, ``server`` if it originated from the server, or ``unknown`` if it's origin isn't known.
+- ``reporter`` identifies the origin of the request/response. It is set to ``client`` if it originated from the client, ``server`` if it originated from the server, or ``unknown`` if its origin is unknown.
+
+Options
+"""""""
+
+This metric supports :ref:`Context Options<hubble_context_options>`.
+
+``httpV2``
+~~~~~~~~~~
+
+``httpV2`` is an updated version of the existing ``http`` metrics.
+These metrics should not be enabled at the same time as ``http``.
```

Comment:
From what I see, if both are enabled then only `httpV2` metrics are exposed? I guess it's the best option (as `http_request_duration_seconds` has the same labels), but would be nice to document that, also in the upgrade notes - the migration path is a little tricky if you can't have both old and new at the same time.

Reply:
My understanding is that it is not possible to enable both `http` and `httpV2`, the second commit in the stack ("hubble: Ensure http and httpV2 metrics cannot both be enabled") implement the conflict detection.

Admittedly, the doc could be clearer about the incompatibility (i.e. state "_can not_ be enabled at the same time […]" rather than "_should not_ be enabled at the same time […]". Does that make sense?

Reply:
Yeah, makes sense to me. My worry here was mainly about the migration path - if we say both old and new "should not" be enabled at the same time, it might be tempting to enable both for a short period, then migrate daashboards, alerts, etc, then disable old. But it's not possible.